src/nupic/algorithms/Connections.hpp

```diff
@@ -402,6 +402,15 @@ class Connections : public Serializable
    * @param inputVector  An SDR
    * @param increment  Change in permanence for synapses with active presynapses.
    * @param decrement  Change in permanence for synapses with inactive presynapses.
+   *
+   * @params previousUpdates & currentUpdates contain the permanence changes done
+   * by this method.  AdaptSegment will not  apply the same learning update to a
```

Comment:
please document more and separate, from what I understand: 
- currentUpdates is the output vector, it gets filled by synapse permanences learned in this method, 
- prevUpdates are currentUpdates from T-1, unchanged, only as input here